GVFS/GVFS.UnitTests.Windows/Windows/Virtualization/WindowsFileSystemVirtualizerTests.cs

```diff
@@ -526,7 +529,13 @@ public void MoveFileIntoDotGitDirectory()
                     fileSystemCallbacks.TryStart(out string error).ShouldEqual(true);
 
                     NotificationType notificationType = NotificationType.UseExistingMask;
-                    mockVirtualization.OnNotifyFileRenamed("test.txt", Path.Combine(".git", "test.txt"), isDirectory: false, notificationMask: ref notificationType);
+                    mockVirtualization.OnNotifyFileRenamed(
+                        "test.txt",
+                        Path.Combine(".git", "test.txt"),
+                        isDirectory: false,
+                        triggeringProcessId: TriggeringProcessId,
+                        triggeringProcessImageFileName: TriggeringProcessImageFileName,
+                        notificationMask: out notificationType);
                     fileSystemCallbacks.OnIndexFileChangeCallCount.ShouldEqual(0);

```

Comment:
Add validation of `notificationType`, e.g. `notificationType.ShouldEqual(NotificationType.UseExistingMask)`?